.github/workflows/build-image.yml

```
@@ -0,0 +1,72 @@
+name: Re-build and publish key4hep-externals image
+on:
+  push:
+    branches:
+    - main
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write
+  attestations: write
+
+jobs:
+  build-externals:
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.image }}-${{ matrix.build_type }}
```

```suggestion
      group: ${{ github.workflow }}-${{ github.ref }}
```

I think I suggested this that I copied and pasted from somewhere else, but here there isn't a `matrix.image` nor a `matrix.build_type`. This should be enough to make it work since it will use the name of the workflow, but we only want one to run at any given time.